EasyPost/Services/Beta/ReferralService.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost._base;
+using EasyPost.Models.API;
+using EasyPost.Models.API.Beta;
+using RestSharp;
+
+namespace EasyPost.Services.Beta
+{
+    public class ReferralService : EasyPostService
+    {
+        public ReferralService(EasyPostClient client)
+            : base(client)
+        {
+        }
+
+        #region CRUD Operations
+
+        /// <summary>
+        ///     Add a Stripe payment method to a Referral Customer.
+        ///     This function should be called against a Client configured with the Referral Customer's API key.
+        ///     This function will fail if called against a non-Referral Customer Client.
+        /// </summary>
+        /// <param name="stripeCustomerId">Unique customer ID provided by Stripe.</param>
+        /// <param name="paymentMethodReference">ID of the card or bank account provided by Stripe.</param>
+        /// <param name="priority">Which priority to save this payment method as on EasyPost.</param>
+        /// <returns>A <see cref="PaymentMethod"/> object.</returns>
+        /// <exception cref="Exception">When the request fails.</exception>
+        public async Task<PaymentMethod> AddPaymentMethod(string stripeCustomerId, string paymentMethodReference, PaymentMethod.Priority? priority = null)
```

Comment:
```suggestion
        public async Task<PaymentMethod> AddPaymentMethod(string stripeCustomerId, string paymentMethodReference, PaymentMethod.Priority? primaryOrSecondary = null)
```

This should be called `primaryOrSecondary`

Reply:
Would introduce either a breaking change or a mismatch since it's called "priority" in the other functions that use this. We can revisit this later.

Reply:
This definitely should have been done initially. This is now the outlier lib and won't match our documentation.

Reply:
The C# example on our docs page is correct.

Reply:
We have the table above it that it won't match. There's also some incoming docs changes that will qualify my statement.

Reply:
Moving conversation internal.